components/page-header.js

```diff
@@ -0,0 +1,11 @@
+class PageHeader {
+  constructor(headerElement){
+    this.headerElement = headerElement;
+  }
+  updateAverage(newAverage) {
+    console.log('newAverage:', newAverage)
+    var badgeElement = this.headerElement.querySelector('span');
+    var displayedAvg = badgeElement.textContent = newAverage
+    console.log(displayedAvg)
```

Comment:
Please remove these `console.log`s now that you've verified that updating the average works.